packages/abstractions/src/multipartBody.ts

```diff
@@ -26,9 +26,10 @@ export class MultipartBody implements Parsable {
 	 * @param partName the name of the part to add or replace.
 	 * @param partContentType the content type of the part to add or replace.
 	 * @param content the content of the part to add or replace.
+	 * @param fileName the name of the file associated with this part.
 	 * @param serializationCallback the serialization callback to use when serializing the part.
 	 */
-	public addOrReplacePart<T>(partName: string, partContentType: string, content: T, serializationCallback?: ModelSerializerFunction<Parsable>): void {
+	public addOrReplacePart<T>(partName: string, partContentType: string, content: T, fileName?: string, serializationCallback?: ModelSerializerFunction<Parsable>): void {
```

Comment:
```suggestion
	public addOrReplacePart<T>(partName: string, partContentType: string, content: T, serializationCallback?: ModelSerializerFunction<Parsable>, fileName?: string): void {
```

To align with the dotnet implementation https://github.com/microsoft/kiota-dotnet/blob/dbbd3d0feddf0aeeb291de22f64f11d7778cd1e6/src/abstractions/MultipartBody.cs#L36